src/review/review.service.ts

```diff
@@ -29,97 +31,116 @@ export class ReviewService {
   ) {}
 
   async createReview(reviewDetails: CreateReviewDto) {
-    const { hotLevel, userId, foodId, tags } = reviewDetails;
-    const review = new Review();
-    const hotLevelId = produceHotLevelId(hotLevel);
-    review.hotLevel = await getRepository(FoodLevel).findOne(hotLevelId);
-    review.user = await getRepository(User).findOne(userId);
-    review.food = await getRepository(Food).findOne(foodId);
-    review.tasteReviews = await getRepository(TasteTag).find({
-      name: In(tags),
-    });
-
-    const result = await this.reviewRepository.save(review);
-
-    return {
-      userId: result.user.id,
-      foodId: result.food.id,
-    };
-  }
-
-  async createReviews(reviewsDetails: CreateReviewsDto) {
-    const { userId, reviewList } = reviewsDetails;
-
-    reviewList.map(async ({ foodId, hotLevel, tags }) => {
+    try{
+      const { hotLevel, userId, foodId, tags } = reviewDetails;
       const review = new Review();
       const hotLevelId = produceHotLevelId(hotLevel);
-
+      review.hotLevel = await getRepository(FoodLevel).findOne(hotLevelId);
       review.user = await getRepository(User).findOne(userId);
       review.food = await getRepository(Food).findOne(foodId);
-      review.hotLevel = await getRepository(FoodLevel).findOne(hotLevelId);
-      review.tasteReviews = await getRepository(TasteTag).find({
-        name: In(tags),
+      review.tasteReviews = await Promise.all(
+        tags.map(async (tagid) => {
+          const tag = getRepository(TasteTag).findOne(tagid);
+          return tag;
+        }),
+      );
+      const result = await this.reviewRepository.save(review);
+      return {
+        userId: result.user.id,
+        foodId: result.food.id,
+      };
+    } catch (e) {
+      throw new HttpException(ERROR_MESSAGE.NOT_FOUND, HttpStatus.BAD_REQUEST);
```

Comment:
```suggestion
      throw new HttpException(ERROR_MESSAGE.BAD_REQUEST, HttpStatus.BAD_REQUEST);
```

리뷰가 만들어지는 API이다보니 `찾으시는 정보나 페이지가 존재하지 않아요!` 이 메세지 보다는 `잘못된 요청입니다. 다시 시도해 주세요!` 값이 맞는 것 같습니다!